UI/refer.csv

```diff
@@ -326,8 +326,8 @@ pso2_players_manual_preparations_quest_arksbingocard#0,"""<url http://pso2.jp/pl
 pso2_players_manual_preparations_loginstamp#0,"""<url http://pso2.jp/players/manual/preparations/info/loginstamp/>"""
 pso2_players_manual_preparations_info_arksroad#0,"""<url http://pso2.jp/players/manual/preparations/info/arksroad/>"""
 pso2_players_manual_preparations_party_supportpartner#0,"""<url http://pso2.jp/players/manual/preparations/party/supportpartner/>"""
-AC_CHARGE_PAGE#0,"""https://AC-CHARGE-URL"""
-USER-INFO-CHANGE-URL#0,"""https://USER-INFO-CHANGE-URL"""
+AC_CHARGE_PAGE#0,"""Https://AC-CHARGE-URL"""
```

Comment:
Why are you changing this URL? why all the https:// ?

Reply:
wtf, i fixed that! Dx